x-pack/legacy/plugins/canvas/canvas_plugin_src/renderers/time_filter/components/pretty_duration/lib/quick_ranges.ts

```diff
@@ -1,53 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License;
- * you may not use this file except in compliance with the Elastic License.
- */
-
-import { UnitStrings } from '../../../../../../i18n/units';
-
-export interface QuickRange {
-  /** Start date string of range */
-  from: string;
-  /** Start date string of range */
-  to: string;
-  /** Display name describing date range */
-  display: string;
-}
-
-const { quickRanges: strings } = UnitStrings;
-
-export const quickRanges: QuickRange[] = [
```

Comment:
Is there any value to keeping these ranges in a lib somewhere? Afaik they were only used in the time filter. 

If not, I should probably remove the associated i18n strings as well.